### PR TITLE
📖  Improve bug reporting clues

### PR DESCRIPTION
**Pull request by MikeSpreitzer**
## Summary
This PR improves bug-reporting clues in two places. One is the GitHub template for a bug-report style of Issue. The other is the "Making a good trouble report" section of the Troubleshooting page of the website.

Preview at https://mikespreitzer.github.io/kcp-edge-mc/doc-report-more/direct/troubleshooting/#making-a-good-trouble-report

## Related issue(s)

Fixes #

### CODE REVIEW

File: docs/content/direct/troubleshooting.md [hunk 1]
```diff
@@ -61,5 +62,13 @@ Show the particulars of something going wrong.
     - In the WDS: the workload objects involved; any `BidingPolicy` involved, and the corresponding `Binding` for each; any `CustomTransform`, `StatusCollector`, or `CombinedStatus` involved.
     - Any involved objects in the WEC(s).
     - Implementation objects in the ITS: `ManifestWork`, `WorkStatus`.
+    - Here is one way to show the evolution of a relevant set of objects over time. The following command displays the `ManifestWork` objects, after creation and after each update (modulo the gaps allowed by eventual consistency), in an ITS as addressed by the kubeconfig context named `its1` --- after first listing the existing objects. Each line is prefixed with the hour:minute:secod at which it appears.
```
Comment (waltforme) [inline docs/content/direct/troubleshooting.md:65]: Typo 'secod'.

Comment (MikeSpreitzer) [reply docs/content/direct/troubleshooting.md]: Thanks, fixed.

**Pull request merged**